learn/tanssi/appchain-services/block-production.md

```diff
@@ -131,18 +106,7 @@ Leveraging this ability to access the states, the current sequencer with the aut
 
 Once the block is filled with appchain transactions, it will be proposed as a candidate and handed over to the Tanssi chain, where the security provider's operators will ensure that the included state proofs match the state proofs from the latest state of Tanssi (preventing unauthorized block production) and that the transactions produced valid state transitions. Having verified the work of the sequencer, the operators will finalize the proposed block, including its candidate receipt in a Tanssi network block.
 
-```mermaid
-classDiagram
-
-    note for tanssi "The sequencer reads the Tanssi State"
-
-    class tanssi["Appchain Block"] {
-        Appchain's Current Sequencers Assignment
-        Tanssi's State Root
-        Sequencer operator's signature
-        Appchain Transactions()
-    }
-```
+![Tanssi-powered network block](/images/learn/tanssi/appchain-services/block-production/block-production-3.webp)
```

Comment:
This image has Polkadot logo etc. I think just doing minor edits will be an easy win

Reply:
Wait a sec, maybe you are seeing the outdated version. This file looks like this:
<img width="1029" alt="image" src="https://github.com/user-attachments/assets/bf5bbb96-1e17-4432-b0b9-46129586e279" />

Reply:
😬